src/chainparams.cpp

```diff
@@ -107,6 +107,8 @@ class CMainParams : public CChainParams {
         pchMessageStart[3] = 0xd9;
         nDefaultPort = 8333;
         nPruneAfterHeight = 100000;
+        m_assumed_blockchain_size = 200;
```

Comment:
This used to be 220?

Reply:
Well spotted, thanks! This was 200 when the PR was made but changed down the line with this maintenance for 0.17: https://github.com/bitcoin/bitcoin/commit/78dae8caccd82cfbfd76557f1fb7d7557c7b5edb. Do you think this should be addressed here or could wait for the next maintenance (since the release process now asks this to be change)?

Reply:
Could bump it to 240 before 0.18 is branched, so we don't forget about it?

Reply:
Yes, thanks. Like this? -> https://github.com/bitcoin/bitcoin/pull/15183